### ISSUE TITLE
threadedselectreactor is made available as though it were a reactor (but it is not a reactor)

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2126]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#2126|
|Type|defect|
|Created|2006-09-25 17:05:54Z|

It shouldn't be in the reactors dictionary, and maybe it shouldn't have "reactor" in its name.

<details><summary>Searchable metadata</summary>

```
trac-id__2126 2126
type__defect defect
reporter__exarkun exarkun
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1159203954000000 1159203954000000
changetime__1159962804000000 1159962804000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun removed owner|
|-|-|

Renamed in obscure-tsr-2126, please review.

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)<a name="note_2"></a>|@jml set owner to @exarkun|
|-|-|

Looks good. Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [18387]) Merge obscure-tsr-2126

Author: exarkun
Reviewer: jml
Fixes [#2126](https://github.com/twisted/twisted/issues/2126)

This renames the ThreadedSelectReactor module and removes its name from the reactors
dictionary so as to indicate that it is not a directly usable reactor.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
